fastestimator/network.py

```diff
@@ -104,7 +105,12 @@ def load_epoch(self, mode: str, epoch: int, output_keys: Optional[Set[str]] = No
         for idx, gradient_op in enumerate(gradient_ops):
             gradient_op.fe_retain_graph(idx != len(gradient_ops) - 1)
         self.epoch_state = {
-            "warmup": warmup, "mode": mode, "req_grad": len(gradient_ops) > 0, "epoch": epoch, "deferred": {}
+            "warmup": warmup,
+            "mode": mode,
+            "req_grad": len(gradient_ops) > 0,
+            "epoch": epoch,
+            "deferred": {},
+            "scaler": self.scaler
```

Comment:
why is scalar being added to base network instead of torch network?

Reply:
it needs to go to state dictionary to be used easily, just like the tf tape